### PR TITLE
Implemented E2E test for product search.

### PR DESCRIPTION
**Pull request by oleksii-kondratiuk**

### CODE REVIEW

File: cypress/integration/e2e/ProductsSearch.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+describe('Products Search Test', function() {
+  it('Skakater Nordmart Store', function() {
+    cy.visit('http://0.0.0.0:4200')
```
Comment (ahmedwaleedmalik) [inline cypress/integration/e2e/ProductsSearch.ts]: It won't be running at this URL, in turn it should use an environment variable. https://docs.cypress.io/guides/guides/environment-variables.html#We-can-move-this-into-a-Cypress-environment-variable

Comment (oleksii-kondratiuk) [reply cypress/integration/e2e/ProductsSearch.ts]: Done

**Comment by stakater-user** [comment]
@oleksii-kondratiuk Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins.release.stakater.com/job/nordmart-apps/job/stakater-nordmart-e2e-tests/job/PR-4/1/) has Failed!

**Comment by stakater-user** [comment]
@oleksii-kondratiuk Yikes! You better fix it before anyone else finds out! [Build 2](https://jenkins.release.stakater.com/job/nordmart-apps/job/stakater-nordmart-e2e-tests/job/PR-4/2/) has Failed!

**Review by ahmedwaleedmalik — APPROVED** [review]

**Comment by stakater-user** [comment]
@oleksii-kondratiuk Yikes! You better fix it before anyone else finds out! [Build 3](https://jenkins.release.stakater.com/job/nordmart-apps/job/stakater-nordmart-e2e-tests/job/PR-4/3/) has Failed!

**Review by kahootali — APPROVED** [review]

**Comment by stakater-user** [comment]
@oleksii-kondratiuk Yikes! You better fix it before anyone else finds out! [Build 4](https://jenkins.release.stakater.com/job/nordmart-apps/job/stakater-nordmart-e2e-tests/job/PR-4/4/) has Failed!

**Pull request closed without merging**